app/helpers/repository_helper.rb

```diff
@@ -1,5 +1,5 @@
 module RepositoryHelper
-  include SubmissionsHelper
+  include FileHelper
```

Comment:
I believe that all files under `app/lib` should be automatically loaded, so I think you should be able to remove this include (and the one below).